docs/UserGuide.md

```diff
@@ -519,6 +519,11 @@ Action | Format | Examples
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous TutorBuddy home folder.
 
+**Q**: May I edit the data file? 
+**A**: Advanced users are welcome to update data directly by editing that data file. AddressBook data are saved as a JSON file `[JAR file location]/data/tutorbuddy.json`.
```

Comment:
Hi! I think it will be good to avoid using "AddressBook data" here as well. Might be better to just omit the Addresbook and put as "Data are saved as...".